crypto/ppccap.c

```diff
@@ -214,6 +214,24 @@ size_t OPENSSL_instrument_bus2(unsigned int *out, size_t cnt, size_t max)
 # endif
 #endif
 
+#if defined(__FreeBSD__)
```

Comment:
Is there a possibility that `OSSL_IMPLEMENT_GETAUXVAL` could already be defined via the block above? And if so, should we have a check for that below so we don't attempt to define it twice?

Reply:
Nope the check above is about Linux/GLIBC cases.